package/package.sh

```diff
@@ -104,6 +104,12 @@ EOF
 
 function _build_graph {
     pushd ${build_dir}
+    enable_breakpad=OFF
+    if [ "x"$build_type = "xDebug" ] || [ "x"$build_type = "xRelWithDebInfo" ]; then
```

Comment:
The breakpad relys on the symbol table, so it only works in Debug/RelWithDebInfo mode. We should not modify this shell script, but the workflow of publish in github. Turn on the breakpad in the publishing workflow.

Reply:
ok.